src/game/AuctionHouse/AuctionHouseMgr.cpp

```diff
@@ -820,8 +820,13 @@ void WorldSession::BuildListAuctionItems(std::vector<AuctionEntry*> const& aucti
             if (itemSubClass != 0xffffffff && proto->SubClass != itemSubClass)
                 continue;
 
-            if (inventoryType != 0xffffffff && proto->InventoryType != inventoryType)
-                continue;
+            if (inventoryType != 0xffffffff && proto->InventoryType != inventoryType) {
+                // if inventory type is chest, we want to return robes too
+                // i.e. cloth chests are in most cases robes by definition
+                if (inventoryType != 5 || proto->InventoryType != 20) {
```

Comment:
There are constants for these values, INVTYPE_ROBE, INVTYPE_CHEST. Use them instead of magic numbers.

Reply:
Didn't know it exists, thanks.